src/ml_gym/frontend/dashboard/src/template/template.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import "./template.css";
+import StackedLineChartIcon from '@mui/icons-material/StackedLineChart';
+import TuneIcon from '@mui/icons-material/Tune';
+import ModelTrainingIcon from '@mui/icons-material/ModelTraining';
+import KeyboardDoubleArrowRightIcon from '@mui/icons-material/KeyboardDoubleArrowRight';
+import KeyboardDoubleArrowLeftIcon from '@mui/icons-material/KeyboardDoubleArrowLeft';
+import CellTowerIcon from '@mui/icons-material/CellTower';
+import { Link } from 'react-router-dom';
+import GlobalConfig from "../features/globalConfig/globalConfig";
+import { FilterConfigType } from "../app/datatypes"
+
+type TemplatePropsType = {
+    children?: React.ReactNode;
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
  children?: React.ReactNode;
```